ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -571,6 +574,7 @@ export default class ConfirmAmendment extends Vue {
     if (draft.error) {
       this.emitError(draft.error)
     } else {
+      this.setUnsavedChanges(false)
```

Comment:
If they are on the confirm page, don't they have unsaved changes?

Reply:
ha - not if it's a draft they loaded in